.idea/runConfigurations/Artemis__Server_.xml

```diff
@@ -1,13 +1,12 @@
 <component name="ProjectRunConfigurationManager">
   <configuration default="false" name="Artemis (Server)" type="SpringBootApplicationConfigurationType" factoryName="Spring Boot">
-    <option name="ACTIVE_PROFILES" value="dev,bamboo,bitbucket,jira,artemis,scheduling,core,local" />
+    <option name="ACTIVE_PROFILES" value="dev,artemis,scheduling,core,local" />
```

Comment:
I guess that change was not intentional, but since we do not use Atlassian products any more it should be fine...